src/components/Select/Select.test.tsx

```diff
@@ -255,4 +256,100 @@ describe("Select", () => {
       "NYPL DS recommends that <select> fields have no more than 10 options; an auto-complete text input is a good alternative for 11 or more options."
     );
   });
+
+  it("Renders the UI snapshot correctly", () => {
+    const siblings = ["Kendall", "Shiv", "Connor", "Roman", "Tom"];
+
+    const primary = renderer
+      .create(
+        <Select
+          id="select"
+          labelText="Which Succession sibling are you?"
+          name="succession-sibling"
+        >
+          {siblings.map((sibling) => (
+            <option>{sibling}</option>
+          ))}
```

Comment:
This gets rendered often. You can define this in a variable and replace it in all the locations where this `.map` gets rendered.